package.json

```diff
@@ -23,16 +24,34 @@
     "build/"
   ],
   "scripts": {
-    "build": "npm run lint && tsc",
-    "lint": "balena-lint --typescript src",
-    "lint-fix": "balena-lint --typescript --fix src",
-    "test": "npm run build",
+    "build": "npm run lint && tsc && rollup -c",
+    "lint": "balena-lint --typescript src tests",
+    "lint-fix": "balena-lint --typescript --fix src tests",
+    "test:node": "mocha -r ts-node/register --reporter spec tests/**/*.spec.ts",
+    "test:browser": "karma start",
+    "test": "npm run build && npm run lint && npm run test:node && npm run test:browser",
```

Comment:
I prefer to put linting last / as a "posttest" because when developing it's nice to be able to run the tests without worrying about linting for fast iteration